docs/admin/upgrading.rst

```diff
@@ -37,9 +37,17 @@ Basic instructions
    Any modifications should have been written within plugins, so that they are not lost on overwriting.
    If this is not the case, take care to maintain your modifications. 
 
-.. note::
+From 1.9 or 1.10 to 1.11
```

Comment:
There have been some problems when skipping minor releases. I don't know if 1.10 can be skipped but I'd rather not encourage it in any way. Saves trouble.